OpenRA.Mods.Common/Traits/ExternalCapturable.cs

```diff
@@ -24,6 +24,9 @@ public class ExternalCapturableInfo : ITraitInfo
 		[Desc("Seconds it takes to change the owner.", "You might want to add a ExternalCapturableBar: trait, too.")]
 		public readonly int CaptureCompleteTime = 15;
 
+		[Desc("Whether autotargeting ignores this actor while it is being captured by an ally.")]
+		public readonly bool AutoTargetIgnores = false;
```

Comment:
I'm with @Mailaender here. If you don't set this to true, this doesn't really fix #6170 (except you add some yaml to the default mods), as the behaviour ingame won't change.